### PR TITLE
New resources

### PR DESCRIPTION
**Pull request by Heismanish**
## What does this PR do?
Added a book resource for kotlin and a completee kotlin programming resource in hindi.

### Why is this valuable (or not)?
This will enable native hindi speakers learn kotlin.

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1328,7 +1328,8 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 ### Kotlin
 
-* [Kotlin Notes for Professionals](https://goalkicker.com/KotlinBook/) - Compiled from StackOverflow documentation (PDF)
+* [Kotlin Head First](https://www.pdfdrive.com/download.pdf?id=196919920&h=774e6318fba6d7ab62fe86c91ee433af&u=cache&ext=pdf) (PDF)
```
Comment (davorpa) [inline books/free-programming-books-langs.md:1331]: Remove non-acceptable content. It has copyright and pdfdrive is not the author site

```suggestion
```

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -1328,7 +1328,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 ### Kotlin
 
-* [Kotlin Notes for Professionals](https://goalkicker.com/KotlinBook/) - Compiled from StackOverflow documentation (PDF)
+* [Kotlin Notes for Professionals](https://goalkicker.com/KotlinBook/) - Compiled from StackOverflow documentation (PDF)
```
Comment (davorpa) [inline books/free-programming-books-langs.md:1331]:
```suggestion
* [Kotlin Notes for Professionals](https://goalkicker.com/KotlinBook/) - Compiled from StackOverflow documentation (PDF)
```

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -18,6 +18,7 @@
     * [Next.js](#nextjs)
     * [Node.js](#nodejs)
     * [React](#react)
+* [Kotlin](#Kotlin)
```
Comment (davorpa) [inline courses/free-courses-hi.md:21]: Fragment ids goes in lowercase

```suggestion
* [Kotlin](#kotlin)
```

File: courses/free-courses-hi.md [hunk 4]
```diff
@@ -189,6 +190,11 @@
 * [ReactJS Tutorials for Beginners In Hindi](https://www.youtube.com/playlist?list=PLUVqY59GNZQNTlOnGne0G7DXnmi7CeOtc) - Pradeep Maurya
 
 
+### Kotlin
+
+* [Kotlin Programming Complete Tutorial in Hindi](https://youtu.be/uoP4JKHgzDE) - Geeky Shows
```
Comment (davorpa) [inline courses/free-courses-hi.md:195]: - Not a playlist but +6h video with chapters integrated in timeline
- Using title as it is
- Expanding YT url
- Adding instructor

```suggestion
* [Kotlin Programming Complete in one Video (Hindi)](https://www.youtube.com/watch?v=uoP4JKHgzDE) - Geeky Shows, `tch.:` Rajesh Kumar
```

**Review by davorpa — CHANGES_REQUESTED** [review]

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**